AllConfigurations leg failing with race condition

It has happened twice on my PR now.

```
12:12:06 Build FAILED.
12:12:06 
12:12:06 D:\j\workspace\windows-TGrou---0d2c9ac4\bin\testPkg\packageTest.targets(104,5): error MSB3371: The file "obj\Debug\\.testComplete" cannot be created. The process cannot access the file 'D:\j\workspace\windows-TGrou---0d2c9ac4\bin\testPkg\projects\System.Composition.Hosting\netcore451\obj\Debug\.testComplete' because it is being used by another process. [D:\j\workspace\windows-TGrou---0d2c9ac4\bin\testPkg\projects\System.Composition.Hosting\netcore451\project.csproj] [D:\j\workspace\windows-TGrou---0d2c9ac4\pkg\test\testPackages.proj]
12:12:06 D:\j\workspace\windows-TGrou---0d2c9ac4\pkg\test\testPackages.proj(177,5): error MSB3073: The command "D:\j\workspace\windows-TGrou---0d2c9ac4\bin/testPkg/tools//dotnet msbuild /t:Test "D:\j\workspace\windows-TGrou---0d2c9ac4\bin/testPkg/test.msbuild" /p:TestPackages=System.Composition.Hosting" exited with code 1.
```

https://ci3.dot.net/job/dotnet_corefx/job/master/job/windows-TGroup_all+CGroup_Debug+AGroup_x64+TestOuter_false_prtest/10842/console

cc: @ericstj @joperezr

--- a/pkg/test/packageTest.targets
+++ b/pkg/test/packageTest.targets
@@ -101,7 +101,7 @@
   <Target Name="_test"
           DependsOnTargets="$(TestDependsOn)">
     <MakeDir Directories="$(IntermediateOutputPath)" />
-    <Touch AlwaysCreate="true" Files="$(IntermediateOutputPath)\.testComplete" />
+    <Touch AlwaysCreate="true" Files="$(IntermediateOutputPath)\$(RuntimeIdentifier)\.testComplete" />
   </Target>
   
   <Target Name="_getTestProjects">

--- a/pkg/test/packageTest.targets
+++ b/pkg/test/packageTest.targets
@@ -101,7 +101,7 @@
   <Target Name="_test"
           DependsOnTargets="$(TestDependsOn)">
     <MakeDir Directories="$(IntermediateOutputPath)" />
-    <Touch AlwaysCreate="true" Files="$(IntermediateOutputPath)\$(RuntimeIdentifier)\.testComplete" />
+    <Touch AlwaysCreate="true" Files="$(IntermediateOutputPath)\$(RuntimeIdentifier).testComplete" />
   </Target>
   
   <Target Name="_getTestProjects">
@@ -113,7 +113,7 @@
         <Semaphore>$(IntermediateOutputPath)\.testComplete</Semaphore>
       </TestProject>
       <TestProject Include="$(MSBuildProjectFullPath)">
-        <Semaphore>$(IntermediateOutputPath)\%(_projectRuntime.Identity)\.testComplete</Semaphore>
+        <Semaphore>$(IntermediateOutputPath)\%(_projectRuntime.Identity).testComplete</Semaphore>
         <AdditionalProperties>RuntimeIdentifier=%(_projectRuntime.Identity)</AdditionalProperties>
       </TestProject>
     </ItemGroup>

